.github/workflows/build.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches: [ '**' ]
     tags-ignore: [ '**' ]
+  pull_request: 
```

Comment:
I didn't understand the purpose of adding the condition for pull requests since the workflow is triggered on all pushes anyway

Reply:
This is for cases like this : https://github.com/aneoconsulting/ArmoniK.Extensions.Csharp/pull/147 where the PR comes from a fork, and thus no push is done on the repo itself, not triggering the CI

Reply:
Yep, exactly